src/QueryBuilder/AbstractDQLQueryBuilder.php

```diff
@@ -509,6 +517,7 @@ protected function defaultConditionClasses(): array
             'NOT' => Condition\Not::class,
             'AND' => Condition\AndX::class,
             'OR' => Condition\OrX::class,
+            'EQUALS' => Condition\Equals::class,
```

Comment:
```suggestion
            '=' => Condition\Equals::class,
```

Maybe replace it with `=`